config/application.rb

```diff
@@ -23,6 +23,8 @@ class Application < Rails::Application
 
     # Custom directories with classes and modules you want to be autoloadable.
     config.autoload_paths += %W(#{config.root}/vendor)
+    config.autoload_paths += %W(#{config.root}/lib/avalon)
+    config.autoload_paths += %W(#{config.root}/lib/avalon/batch)
```

Comment:
I suspect that I don't need these at all. The reason why I added them was because my changes in development weren't showing up when I tested via the browser. I think the majority of the problem is that since most operations do through delayed_job, I needed to restart that service to see changes.

I will remove these paths and the redundant ones you mentioned.